docs/src/dataDisplay/personPanelControlledAccordionExample.jsx

```diff
@@ -0,0 +1,176 @@
+                    case 1:
+                        type = 'home';
+
+                        break;
+                    case 2:
+                        type = 'work';
+
+                        break;
+                    case 3:
+                        type = 'cell';
+
+                        break;
+                    default:
+                }
+
+                return {
+                    type,
+                    isPrimary: phone.isPrimary,
+                    value: phone.displayPhoneNumber,
+                };
+            });
+            const primaryEmergencyContact = find(person.emergencyContacts, 'isPrimary');
+            let recordType;
+
+            if (person.isChild) {
+                recordType = 'child';
+            } else if (person.isStudent) {
+                recordType = 'student';
+            } else {
+                recordType = 'adult';
+            }
+
+            return {
+                personId: person.id,
+                details: {
+                    addresses,
+                    allergies: person.allergies,
+                    birthdate: person.birthDate,
+                    campus: person.churchEntityName,
+                    commonlyAttendedService: person.commonlyAttendedService,
+                    emails,
+                    emergencyContactAddresses:
+                        primaryEmergencyContact && primaryEmergencyContact.addresses,
+                    emergencyContactEmails:
+                        primaryEmergencyContact && primaryEmergencyContact.emails,
+                    emergencyContactName:
+                        primaryEmergencyContact && `${primaryEmergencyContact.firstName} ${primaryEmergencyContact.lastName}`,
+                    emergencyContactPhones:
+                        primaryEmergencyContact && primaryEmergencyContact.phones,
+                    emergencyContactPreferMethod:
+                        primaryEmergencyContact && primaryEmergencyContact.preferredMethod,
+                    emergencyContactRelation:
+                        primaryEmergencyContact && primaryEmergencyContact.relationshipName,
+                    gender: person.gender,
+                    gradeLevel: person.gradeLevel,
+                    isDoNotContact: person.contactPreferences.doNotContact,
+                    phones,
+                    preferredService: person.preferredService,
+                    recordType,
+                },
+                summary: {
+                    avatar: person.profilePictureUrl,
+                    birthdate: person.birthDate,
+                    campus: person.churchEntityName,
+                    emails,
+                    firstName: person.firstName,
+                    gender: person.gender,
+                    gradeLevel: person.gradeLevel,
+                    isDoNotContact: person.contactPreferences.doNotContact,
+                    isDoNotEmail: person.contactPreferences.doNotEmail,
```

Comment:
For a person summary, we might need to make adjustment for record type `child`. We need to show the contact preferences of their primary emergency contact.

Reply:
But still, we can get primary ermergency contact info's like `(isDoNotContact, isDoNotEmail & preferredContactMethod etc)` infos and then passed it to person summary.

Reply:
Is there anything that needs to be done here @fantaJinMode

Reply:
It seems, all good.